docs/features/deserializing-events.md

```diff
@@ -5,6 +5,57 @@ layout: default
 
 ## Deserializing Events
 
+The `Arcus.EventGrid` package provides several ways to deserializing events.
+Following paragraphs describe each supported type of event.
+
+- [Deserializing EventGrid Events](#deserializing-eventgrid-events)
```

Comment:
This is the official name while `CloudEvents` is the official name for that spec
```suggestion
- [Deserializing Event Grid Events](#deserializing-event-grid-events)
```